src/plugins/home/server/services/tutorials/lib/tutorial_schema.ts

```diff
@@ -154,6 +154,9 @@ export const tutorialSchema = schema.object({
   savedObjects: schema.maybe(schema.arrayOf(schema.any())),
   savedObjectsInstallMsg: schema.maybe(schema.string()),
   customStatusCheckName: schema.maybe(schema.string()),
+
+  integrationBrowserCategories: schema.maybe(schema.arrayOf(schema.string())),
+  eprPackageOverlap: schema.maybe(schema.string()),
```

Comment:
Is the idea that for Beats modules in the next PR that this field would default to `moduleName` for tutorials that specify one?

Reply:
yes. that was the idea. I will remove this for this PR and introduce when we add the replace-able logic in a follow-up PR.